dataplane/dplanerc/interface.go

```diff
@@ -404,6 +404,17 @@ func (ni *Reconciler) addLAGMember(ctx context.Context, intf ocInterface, member
 		if err := ni.ifaceMgr.LinkSetDown(memberLink); err != nil {
 			log.Warningf("failed to set link %v down: %v", intf.name, err)
 		}
+		for i := 0; i < 3; i++ { // Ensure link is down before continuing.
```

Comment:
How is 3 selected? can we make it a constant?

Reply:
make a const